### ISSUE TITLE
Improve documentation and versioning

### ISSUE THREAD

**Issue by nardeas** [kaizhu256/node-uglifyjs-lite#1]
Great project but documentation is hard to follow. Also why not using semver?

**Comment by kaizhu256** [comment]
sorry for late response, but hopefully these will address your concerns ^^;;;

> documentation is hard to follow
this package is derived from uglifyjs v1.3.5 (the last uglifyjs v1 release), and will stay frozen at that version.  the original source code is only 2 files, which u can use for reference:
https://github.com/mishoo/UglifyJS/blob/v1.3.5/lib/parse-js.js
https://github.com/mishoo/UglifyJS/blob/v1.3.5/lib/process.js

for personal reasons, i want to keep this package as small as possible, so it will stay minified for forseeable future, but i'll add a link in README.md to the original source-files.

the documentation and screen-captures are all auto-generated during the travis-ci build-process (like in all my other packages), and its not worth the effort for me to start manually editing them.

if you want documentation on the latest versions of uglifyjs, i've auto-generated them as well at:
https://www.npmjs.com/package/npmdoc-uglifyjs
and
https://www.npmjs.com/package/npmdoc-uglify-js

> Also why not using semver?
tbh, i have ocd and tend to drastically rewrite code between minor versions, so semver is meaningless as a stability guidance for most of my npm packages (though i've been trying to get better at this)  ^^;;;  also for me personally, prefere knowing the release dates rather than version numbers for software that i use,  as it tells me if the software is stale or not, given how chaotic the javascript world is currently.

**Comment by kaizhu256** [comment]
closing this issue due to inactivity.
also note apidocs are now available @
https://github.com/npmdoc/node-npmdoc-uglifyjs
https://github.com/npmdoc/node-npmdoc-uglify-js
